app.js

```diff
@@ -1,98 +1,307 @@
+    );
+
     for (const identifiersCoupleOP of identifiersCouplesOP) {
       const wegwijsOvo = identifiersCouplesWegwijs[identifiersCoupleOP.kbo];
       // If a KBO can't be found in wegwijs but we already have an OVO for it in OP, we keep that OVO.
       // It happens especially a lot for worship services that sometimes lack data in Wegwijs
-      if (wegwijsOvo && (identifiersCoupleOP.ovo != wegwijsOvo)) {
+      if (wegwijsOvo && identifiersCoupleOP.ovo != wegwijsOvo) {
         // We have a mismatch, let's update the OVO number
         let ovoStructuredIdUri = identifiersCoupleOP.ovoStructuredId;
         if (!ovoStructuredIdUri) {
-          ovoStructuredIdUri = await constructOvoStructure(identifiersCoupleOP.kboStructuredId);
+          ovoStructuredIdUri = await constructOvoStructure(
+            identifiersCoupleOP.kboStructuredId
+          );
         }
 
         await updateOvoNumberAndUri(ovoStructuredIdUri, wegwijsOvo);
       }
     }
-    console.log('Healing complete!');
+    console.log("Healing complete!");
   } catch (err) {
-    console.log(`An error occurred during OVO numbers healing: ${err}`)
+    console.log(`An error occurred during OVO numbers healing: ${err}`);
   }
 }
 
-async function getAllOvoAndKboCouplesWegwijs() {
+async function healKboUnits() {
+  try {
+    console.log("Healing kbo starting...");
+    const kboIdentifiersOP = await getAllAbbKboOrganizations();
+    const kboIdentifiersWegwijs = await getAllOvoAndKboCouplesWegwijs(true);
+
+    let update = false;
+
+    for (const kboIdentifierOP of kboIdentifiersOP) {
+      const wegwijsKboOrg = kboIdentifiersWegwijs[kboIdentifierOP.kbo];
+
+      if (wegwijsKboOrg) {
+        if (!kboIdentifierOP.kboOrg) {
+
+          let newKboOrgUri = await createNewKboOrg(wegwijsKboOrg, kboIdentifierOP.kboId);
+          await linkAbbOrgToKboOrg(kboIdentifierOP.abbOrg, newKboOrgUri);
+        }
+
+        if (kboIdentifierOP.changeTime) {
+          update = new Date(kboObject.changeTime).getTime() > new Date(kboIdentifiers.changeTime).getTime() ? true : false;
+        }
+
+        if (update) {
+          const kboIdentifiers = await getKboIdentifiers(identifiers.adminUnit);
+          updateKboOrg(wegwijsKboOrg, kboIdentifiers);
+        }
+      }
+    }
+    console.log("Healing complete!");
+  } catch (err) {
+    console.log(`An error occurred during KBO organization healing: ${err}`);
+  }
+}
+
+async function getAllOvoAndKboCouplesWegwijs(healKboUnit) {
   let couples = {};
 
-  const response = await fetch(`https://api.wegwijs.vlaanderen.be/v1/search/organisations?q=kboNumber:/.*[0-9].*/&fields=ovoNumber,kboNumber&scroll=true`);
-  const scrollId = JSON.parse(response.headers.get('x-search-metadata')).scrollId;
+  const response = await fetch(
+    `https://api.wegwijs.vlaanderen.be/v1/search/organisations?q=kboNumber:/.*[0-9].*/&changeTime,name,shortName,ovoNumber,kboNumber,labels,contacts,organisationClassifications,locations,parents&scroll=true`
```

Comment:
**Issue:** `fields=` is missing in the url. 

Moreover, it will be safer and more maintainable to declare the list of fields once in the top of the file and reuse it each time is needed, than have it duplicate in many places.